### DISCUSSION TITLE
Not cant Import

### DISCUSSION THREAD

**Discussion by EduirBR** [gopxl/pixel#79] [Q&A]
Hello Pixel mates, I did a demo game using the previous version of pixel but now I cant use the old version or the new one.

error while importing github.com/faiface/pixel/pixelgl: build constraints exclude all Go files, I formated my pc since last time I was using that repo but now looks like its doent works and I've used fiber and libraries like that for API and all good the only bad its trying to get pixelgl from original repo or from the v1 or opengl on the v2

**Comment by xinkiknix** [comment]
I have a similar issue: I get following error messages when attempting to compile any project using gopxl:
`imports github.com/gopxl/pixel/pixelgl
	imports github.com/go-gl/gl/v3.3-core/gl: build constraints exclude all Go files in C:\Users\ADMIN\go\pkg\mod\github.com\go-gl\gl@v0.0.0-20211210172815-726fda9656d6\v3.3-core\gl`

I created a new project using window example and get same compile errors. did run go module tidy, vendor, ... go get -u -v,

**Comment by dusk125** [comment]
> Hello Pixel mates, I did a demo game using the previous version of pixel but now I cant use the old version or the new one.
> 
> error while importing github.com/faiface/pixel/pixelgl: build constraints exclude all Go files, I formated my pc since last time I was using that repo but now looks like its doent works and I've used fiber and libraries like that for API and all good the only bad its trying to get pixelgl from original repo or from the v1 or opengl on the v2

Odd, the old project is still there, but if you want to use the new version `github.com/gopxl/pixel/v2` and `github.com/gopxl/pixel/v2/backends/opengl`
